wire-ios/Wire-iOS/Sources/UserInterface/Helpers/UIViewController+SafeArea.swift

```diff
@@ -18,66 +18,7 @@
 
 import UIKit
 
-// TODO: [WPB-8778] remove the whole file
```

Comment:
Can't we delete the file as well?

Reply:
Done: https://github.com/wireapp/wire-ios/pull/1907/commits/c694ecbecc1b5698be694c4d1883023fd806845d